src/Fabulous.Avalonia/Views/_StyledElement.fs

```diff
@@ -61,6 +61,11 @@ module StyledElement =
                 style.Source <- Uri(value)
                 styles.Add(style))
 
+    /// Allows adding inline styles to a StyledElement.
+    let InlineStyles =
+        Attributes.defineProperty "StyledElement_InlineStyles" Unchecked.defaultof<IStyle seq> (fun target values ->
```

Comment:
I think would be better to use `definePropertyWithGetSet` here as we can then set a proper setter value instead of `Unchecked.defaultof<IStyle seq>` e.g.
```fsharp
    let InlineStyles =
        Attributes.definePropertyWithGetSet<IStyle seq> "StyledElement_Styles" (fun target -> (target :?> StyledElement).Styles) (fun target styles ->
            let target = (target :?> StyledElement)
            target.Styles.Clear()

            for style in styles do
                target.Styles.Add(style))
```

Reply:
I don't know whether I want to **replace** instead of **add to the target styles** in every case. In my examples adding to any pre-existing styles works fine and I haven't tried replacing - but I can imagine a scenario in which you'd want to do that.

Reply:
The reason for `target.Styles.Clear()` is to avoid having duplicated styles.